accordproject/vscode-web-extension
# feat(codeactions): add code actions for updating namespace and voc validation

### Changes
- Enables map support and import aliasing in CTO files
- Code action to update the namespace in a model file, updating all imports and vocabulary files
- Find references to find all declarations that reference a concept
- Validation of vocabulary files
- Update VSCode version and other dependencies
- Check that extension runs locally in chrome using `npm run chrome`
- Add a stub unit test

### Flags
- See code comments for limitations when editing voc and cto files.

### Screenshots or Video


### Related Issues
- Issue #<NUMBER>
- Pull Request #<NUMBER>

### Author Checklist
- [ ] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [ ] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [ ] Merging to `master` from `fork:branchname`
